ansible/deploy.sh

```diff
@@ -40,6 +40,10 @@ case $key in
     GITHUB_PASSWORD="$2"
     shift
```

Comment:
please add flag description to help output here https://github.com/shadow-robot/sr-build-tools/blob/F_deploy_secure/ansible/deploy.sh#L98